src/LEGO.AsyncAPI/LEGO.AsyncAPI.csproj

```diff
@@ -27,6 +19,7 @@
       <PrivateAssets>all</PrivateAssets>
       <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
     </PackageReference>
+    <PackageReference Include="System.Text.Json" Version="8.0.2" />
```

Comment:
This library also has to be included, since it does not exist out of the box in netstandard.